lib/logstash/outputs/elasticsearch/ilm.rb

```diff
@@ -1,116 +1,7 @@
 module LogStash; module Outputs; class ElasticSearch
   module Ilm
-
```

Comment:
Cant this file be removed? I thought ilm is xpack

Reply:
Ack